install/kubernetes/tetragon/templates/_container_rthooks.tpl

```diff
@@ -16,6 +16,14 @@
     - hook-args
     - --fail-allow-namespaces
     - {{ if .Values.rthooks.failAllowNamespaces }}{{ printf "%s,%s" .Release.Namespace .Values.rthooks.failAllowNamespaces }}{{ else }}{{ .Release.Namespace }}{{ end }}
+   {{- range $key, $value := .Values.rthooks.extraHookArgs }}
+   {{- if $value }}
```

Comment:
`{{- if eq nil $value }}` would be better I think, to distinguish between `$value` being null or empty string/array/object.

Reply:
Fixed, thanks!